masterlist.yaml

```
@@ -24105,6 +24105,11 @@ plugins:
     inc: [ 'Paper World Map Unofficial Fix.esp' ]
     req: [ *FlatMapMarkers ]
 
+  - name: 'FWMF for Paper Map by Mirhayasu.esp'
+    group: 'Worldspace Settings'
+    url:
+      - 'https://www.nexusmods.com/skyrimspecialedition/mods/29932'
```

Needs a few format changes.

```suggestion
  - name: 'FWMF for Paper Map by Mirhayasu.esp'
    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/29932/' ]
    group: *worldSettingsGroup
```